Makefile

```diff
@@ -0,0 +1,5 @@
+run: build
+	docker run -it --rm --volume=${CURDIR}/src/main/resources:/app/src/main/resources:Z resy-booking-bot
```

Comment:
When we run it we mount the `src/main/resources` folder into the container so that local configuration changes are reflected. The `:Z` is relevant to systems running SELinux and `podman`, I'm slightly concerned it won't work on other boxes.

Reply:
I could test this out on my machine and see if it breaks. I want to try it out regardless just to see if it even works at all for me.